scripts/taurusdoc

```diff
@@ -79,7 +80,7 @@ class RstCreator(auto_rst4api.Auto_rst4API_Creator):
         ofname = os.path.join(docparentpath, "_%s.rst" % classname)
 
         if self.verbose:
-            print 'creating "%s" ...' % ofname,
+            print('creating "%s" ...' % ofname,
```

Comment:
This change is definitely wrong. The parenthesis should close in this same line, not after the "info" declaration

Reply:
Never mind for one print, we will see after the review of the code.
Right now I am working on porting the code with futurize and resolving issues, my goal is to pass all tests.
After that we can prepare a python3 Travis CI to start testing with both python2 and 3.

Reply:
But this print is important because if the change was done by futurize automatically, it means that we should pay a lot of attention to that kind of changes (print statements ending with a comma). Note that the resulting code is completely wrong

Reply:
I Will check more deeply